dep/conda/Makefile

```diff
@@ -1,8 +1,8 @@
 # Build conda environments
 
 # Define
-MINICONDA_INSTALLER = Miniconda3-4.5.11-Linux-x86_64.sh
-
+MINICONDA_INSTALLER = Miniconda3-4.7.12-Linux-x86_64.sh
+SHELL := /bin/bash
```

Comment:
The default shell (without this line) sometimes is not bash, and `source` (used below in this makefile) is not available.